<launch>
	<arg name="joy" default="true"/>

	<group if="$(arg joy)">
		<node pkg="joy" type="joy_node" name="joy">
			<param name="dev" value="/dev/input/js0"/>
			<param name="autorepeat_rate" value="10.0"/>
		</node>

		<node pkg="teleop_twist_joy" type="teleop_node" name="teleop_joy" output="screen">
			<remap from="cmd_vel" to="/manual_cmd_vel"/>
			<param name="scale_linear" value="1.0"/>
			<param name="scale_angular" value="1.0"/>
		</node>
	</group>

	<node pkg="topic_tools" type="mux" name="cmd_vel_mux" args="cmd_vel auto_cmd_vel manual_cmd_vel estop_cmd_vel mux:=mux_cmd_vel"/>

	<node pkg="pwm_dev" type="select_mode.py" name="select_mode"></node>

	<include file="$(find realsense2_camera)/launch/rs_rgbd.launch"/>

	<node pkg="pwm_dev" type="zero_twist.py" name="estop"></node>
</launch>
